decidim-core/spec/factory_lint_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Lint factories", processing_uploads_for: Decidim::AttachmentUploader do
+  it "has 100% valid factories" do
+    expect { FactoryGirl.lint }.not_to raise_error
```

Comment:
Can you add a `traits: true` option to the `lint` method, so that we also check for traits?

Source: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories